pkg/agent/tasks/lib/scenedetection/svm_poly2.py

```diff
@@ -440,6 +440,110 @@ def _enumerate_scene_candidates(result_queue, args):
 
     return result
 
+def find_match(curr, ref, width, height, index):
+        # Defining region of search
+        img_x = int(width/6)
+        img_y = int(height/6)
+        img_w = int(2 * width/3)
+        img_h = int(2 * height/3)
+
+        # Create target
+        crop_image = ref[img_y:img_y+img_h, img_x:img_x+img_w]
+
+        # Initiate SIFT detector
+        sift = cv2.SIFT_create()
+
+        # Find the keypoints and descriptors with SIFT
+        kp1, des1 = sift.detectAndCompute(crop_image, None)
+        kp2, des2 = sift.detectAndCompute(curr, None)
+
+        # If no features were found in either image, skip
+        if des1 is None or des2 is None or len(kp1) < 2 or len(kp2) < 2:
+            # print(f"No descriptors found (des1={des1 is None}, des2={des2 is None}) at index {index}")
+            return False
+
+        # FLANN parameters
+        FLANN_INDEX_KDTREE = 1
+        index_params = dict(algorithm = FLANN_INDEX_KDTREE, trees = 5)
+        search_params = dict(checks = 50)
+
+        flann = cv2.FlannBasedMatcher(index_params, search_params)
+
+        matches = flann.knnMatch(des1, des2, k=2)
+
+        # Ratio test as per Lowe's paper
+        good = []
+        for i,(m,n) in enumerate(matches):
+            if m.distance < 0.7 * n.distance:
+                good.append(m)
+
+        if len(good) < 100:
```

Comment:
Arbtirary constant should be in a variable. Documentation... Valid range? Web resources? When should it be increased? decreased?